home.html

```diff
@@ -746,6 +871,18 @@ <h3 class="h3">Lorem ipsum dolor sit amet, consectetur adipisicing.</h3>
           <h2 class="h2">Lorem ipsum dolor sit amet, consectetur adipisicing.</h2>
           <h1 class="h1">Lorem ipsum dolor sit amet, consectetur adipisicing.</h1>
           <div class="hero">Lorem ipsum dolor sit amet, consectetur adipisicing.</div>
+          
+          <pre>
+          <code class="langauge-html">
+            <div class="text-center">lorem centered text</div>
+            <div class="text-sm">Small Text</div>
+            <div class="text-grey">example for grey text</div>
+          </code>
+          </pre>
+
+          <div class="text-center">lorem centered text</div>
+          <div class="text-sm">Small Text</div>
+          <div class="text-grey">example for grey text</div>
       </main>
     </article>
```

Comment:
just check the icons of component card with badges, rest of the code looks clean  and good, you can proceed to merge...